### DISCUSSION TITLE
Carreiras "Legado" em 2024

### DISCUSSION THREAD

**Discussion by rdgspc** [balta-io/forum#473] [Tira Dúvidas] [answered]
Balta, boa tarde!
Primeiramente meus parabéns pela qualidade e didática do .net fundamentos.
Desejo dar continuidade no aprendizado. Posso seguir a trilha backend .net "legado" normalmente?
E você acredita que dominando os módulos, teria aptidão para tentar vaga junior? Me parede um bom roadmap.

**Comment by andrebaltieri** [comment, marked as answer]
> Balta, boa tarde!
> 
> Primeiramente meus parabéns pela qualidade e didática do .net fundamentos.
> 
> Desejo dar continuidade no aprendizado. Posso seguir a trilha backend .net "legado" normalmente?
> 
> E você acredita que dominando os módulos, teria aptidão para tentar vaga junior? Me parede um bom roadmap. 

Boa tarde, @rdgspc, como vai?

Muito obrigado pelo feedback positivo, fico feliz que curtiu a didática!

Sobre o conteúdo, nossas carreiras estão mais atualizadas que esta que citou, é só seguir na ordem que é apresentado nesta página:
https://balta.io/carreiras

Qualquer dúvida só chamar!

Bons estudos 🚀
